DatabaseStructures

```diff
@@ -0,0 +1,26 @@
+PHONE
```

Comment:
Maybe we should add screen resolution as a parameter too?

Reply:
Ok